### ISSUE TITLE
Last line of conditional assignment

### ISSUE THREAD

**Issue by lewis6991** [vhda/verilog_systemverilog.vim#84]
Current indent behaviour:

``` systemverilog
assign out = cond0 ? a :
             cond1 ? b :
             c;
```

Proposed indent behaviour:

``` systemverilog
assign out = cond0 ? a :
             cond1 ? b :
                     c ;
```

@vhda, what do you think about this.

**Comment by vhda** [comment]
Personally I like it. But I propose that we add an option for it because it's possible that this goes against some coding guidelines.

**Comment by lewis6991** [comment]
I agree. Lets add it to `g:verilog_disable_indent` as `conditional`.

**Comment by vhda** [comment]
That's not ideal. For example, the coding guidelines I use require the assignment indentation but don't include the extra indentation you are now proposing. I can always manually delete the extra indent, but I would prefer a second variable.

(Sorry for the extra work)

**Comment by lewis6991** [comment]
I'm proposing that `g:verilog_disable_indent='conditional'` simple disables the indent I want to implement, not assignment indents. The current behaviour makes no extra effort for conditionals.

Its important to differentiate the difference between `assignments` and `conditional assignments`.

**Comment by vhda** [comment]
I misunderstood what you said. That seems perfect.

(I have a release this week... so I'm not really focused here, sorry).

**Comment by vhda** [comment]
Just looking at bad code: take into account nested conditionals.

**Comment by lewis6991** [comment]
Example?

**Comment by vhda** [comment]
@lewis6991 I'm really sorry. Again, I did not look at this closely enough. Your original example already included nested conditionals. Hopefully, starting next week I'll be a bit more focused.

**Comment by lewis6991** [comment]
No worries :)

**Comment by lewis6991** [comment]
Fixed in #87
